src/index.js

```diff
@@ -0,0 +1,3 @@
+import StockX from './apis/stockx';
+
+export { StockX };
```

Comment:
We could change this to a default export if you want. I don't know your plans for this repository, but if you wanted to do more than just a StockX API, you could export more than just StockX here.

I'm all for moving to a default export right now though, as that's the current direction of the API.